src/python/WMCore/ReqMgr/Service/Request.py

```diff
@@ -429,6 +429,12 @@ def _handleAssignmentStateTransition(self, workload, request_args, dn):
         # by default, it contains all unmerged LFNs (used by sites to protect the unmerged area)
         request_args['OutputModulesLFNBases'] = workload.listAllOutputModulesLFNBases()
 
+        # Add parentage relation for step chain, task chain:
+        stepMap = workload.getStepParentageSimpleMapping()
+        if stepMap:
+            request_args["StepChainParentageMap"] = stepMap
```

Comment:
how about just "StepParentageMap"? Then we should define this parameter under the `getWorkloadAssignArgs` method, but instead of using the StdBase class, I'd suggest to define it under the StepChain one, just like I did for the T0 specs:
https://github.com/dmwm/WMCore/pull/8635/files#diff-90f4b259876ad7414f235789db2dbae4R197